### PR TITLE
grow-363 adding tracking links into footer

### PR DESCRIPTION
**Pull request by BoulderBrains**
Tracking events have been added to all clicks within TheFooter.vue based on the structure laid out in grow-363

### CODE REVIEW

**Review by emuvente — APPROVED** [review]
👍

File: src/components/WwwFrame/TheFooter.vue [hunk 1]
```diff
@@ -3,36 +3,58 @@
 		<nav class="small-footer hide-for-large" aria-label="Footer navigation">
 			<ul class="hide-for-print">
 				<li>
-					<router-link :to="applyUrl">
+					<router-link
+						:to="applyUrl"
+						v-kv-track-event="['Footer', 'click-Borrow-mobile view']"
```
Comment (ryan-ludwig) [inline src/components/WwwFrame/TheFooter.vue:8]: My instinct is that we could have the same tracking events for mobile and desktop

Comment (ryan-ludwig) [reply src/components/WwwFrame/TheFooter.vue:8]: Maybe double check with @brandonatkiva

Comment (BoulderBrains) [reply src/components/WwwFrame/TheFooter.vue:8]: You're instincts were right @ryan-ludwig. I connected with Brandon and he requested that the mobile links be mapped to analogous options from the desktop version of the footer. Changes made in latest commit.

**Review by ryan-ludwig — APPROVED** [review]

**Comment by mcstover** [comment]
@BoulderBrains @emuvente These should probably added to styleguide as well so the same events are fired across the entire site.

**Comment by emuvente** [comment]
@mcstover Yeah, I asked Brandon about that. This change is mainly being requested for being able to track interactions on the homepage, so for the sake of reducing the time to make this hotfix we're doing it in the new stack first and in the old stack in a follow-up ticket.

**Pull request merged**